src/client/src/views/SceneForm.js

```diff
@@ -23,8 +22,25 @@ module.exports = {
           },
           value: Scene.current.name ? Scene.current.name : `Scene ${Scene.current.id}`
         }),
-        m("button.save-name-button[type=submit]", "Save")
       ]),
+      m('h2', `Start`),
+      m('form.save-start-form', {
+        onsubmit: function(e) {
+          e.preventDefault()
+        }
+      }, [
+        m("input.new-start-input[type=text]", {
+          oninput: function (e) {
+            Scene.current.start = e.target.value
+          },
+          value: Scene.current.start ? Scene.current.start : ``
+        }),
+      ]),
+      m('button.save-changes-button', {
+        onclick: function() {
```

Comment:
So you are going to want this button to be [type=submit], and no need for a click handler.